codelab-elements/google-codelab-code-menu/google_codelab_code_menu_test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+goog.module('googlecodelabs.CodelabCodeMenuTest');
+goog.setTestOnly();
+
+const CodelabCodeMenu = goog.require('googlecodelabs.CodelabCodeMenu');
+window.customElements.define(CodelabCodeMenu.getTagName(), CodelabCodeMenu);
+const testSuite = goog.require('goog.testing.testSuite');
+goog.require('goog.testing.asserts');
+goog.require('goog.testing.jsunit');
+
+testSuite({
+
+  setUp() {
+  },
+
+  tearDown() {
+  },
+
+  testCodelabSurveyUpgraded() {
```

Comment:
Can we add some basic tests here? Assert that clicking each button does what we expect it to do?